internal/zinc-core/src/main/scala/sbt/internal/inc/Relations.scala

```diff
@@ -139,7 +146,7 @@ trait Relations {
    *
    * This reflects Scala's compiler behavior of generating two class files per top level object declaration.
    */
-  def binaryClassName: Relation[String, String]
+  def productClassName: Relation[String, String]
```

Comment:
I see. You're thinking about the sbt. I narrowed my scope to zinc where I think naming is acceptably consistent. In any case, feel free to keep the new name if it helps you navigating the code.